setup.cfg

```diff
@@ -20,8 +20,8 @@ classifiers =
   Programming Language :: Python
   Programming Language :: Python :: 3
   Programming Language :: Python :: 3 :: Only
```

Comment:
Shouldn't we have a python version specifier in setup.cfg?

Reply:
Like this: https://github.com/sunpy/sunpy/blob/main/setup.cfg#L31

Reply:
Good point I'll add it here